### ISSUE TITLE
mongodbatlas_third_party_integration - microsoft_teams_webhook_url keeps updating on every apply

### ISSUE THREAD

**Issue by ktumu0225** [mongodb/terraform-provider-mongodbatlas#1135]
# Terraform CLI and Terraform MongoDB Atlas Provider Version

```
Terraform v1.4.5
+ provider registry.terraform.io/mongodb/mongodbatlas v1.8.1
```

# Terraform Configuration File
```
resource "mongodbatlas_third_party_integration" "microsoft_teams" {
  project_id                  = mongodbatlas_privatelink_endpoint.default.project_id
  type                        = "MICROSOFT_TEAMS"
  microsoft_teams_webhook_url = var.teams_webhook_url
}
```

# Expected Behavior
No updates on subsequent terraform apply

# Actual Behavior
terraform apply always keeps offering to update the microsoft_teams_webhook_url attribute:

```
  # mongodbatlas_third_party_integration.microsoft_teams will be updated in-place
  ~ resource "mongodbatlas_third_party_integration" "microsoft_teams" {
        id                          = "cHJvamVjdF9pZA==:NjNmOGUyMmM5MTA2NDI2Y2NlODZhODcz-dHlwZQ==:TUlDUk9TT0ZUX1RFQU1T"
      ~ microsoft_teams_webhook_url = (sensitive value)
        # (3 unchanged attributes hidden)
    }
```

# References
This issue looks similar to #963 other than that it is happening for Microsoft Teams Integration now.

**Comment by martinstibbe** [comment]
@ktumu0225 This is a valid bug will be fixed in internal engineering ticket [INTMDB-784](https://jira.mongodb.org/browse/INTMDB-784)

**Comment by Zuhairahmed** [comment]
hi @ktumu0225  we have released fix to this issue as part of v1.10.0 release. can you test and let us know if this issue has been resolved?

**Comment by karthiktumu** [comment]
Yes this issue has been resolved now after upgrading provider to v1.10.0. I just had to apply the change once and the subsequent plans didn't report a resource update.

**Comment by Zuhairahmed** [comment]
wonderful news, thanks @karthiktumu! resolving ticket
